Rego/EXOConfig.rego

```diff
@@ -221,19 +221,19 @@ tests[{
 }
 #--
 
-# Are both the tests supposed to be the same?
-
 #
 # MS.EXO.6.1v1
 #--
 
-SharingPolicyAllowedSharing[SharingPolicy.Name] {
+SharingPolicyContactsAllowedAllDomains[SharingPolicy.Name] {
     SharingPolicy := input.sharing_policy[_]
-    InList := "*" in SharingPolicy.Domains
-    InList == true
+    Domains = SharingPolicy.Domains[_]
+    WildcardInList := contains(Domains, "*")
+    ContactsInList := contains(Domains, "Contacts")
+    WildcardInList == true
+    ContactsInList == true
```

Comment:
Are we doing this compare this way for a reason? 
```
    contains(Domains, "*") == true
    contains(Domains, "Contacts") == true
```

Above would be more concise